hypervisor/boot/sbl/multiboot.c

```diff
@@ -218,6 +218,12 @@ static void *parse_image_boot_params(struct vm *vm, char *cmdline)
 	boot_params->p_platform_info =
 				hpa2gpa(vm, boot_params->p_platform_info);
 
+	if ((boot_params->p_seed_list == INVALID_GPA) || (boot_params->p_platform_info == INVALID_GPA)) {
+		pr_fatal("%s, vm[%d] invalid seed: 0x%llx, or platform: 0x%llx", __func__,
+			vm->vm_id, boot_params->p_seed_list, boot_params->p_platform_info);
+		goto fail;
+	}
+
```

Comment:
This is previous bug?

Reply:
There're lots of code which we didn't check the return value. Maybe we don't have to after we check the whole  logic.